src/main/java/mcp/mobius/betterbarrels/common/items/dolly/ItemBarrelMover.java

```diff
@@ -535,6 +535,21 @@ private boolean isTEMovable(TileEntity te) {
         return false;
     }
 
+    private static boolean tileIsASpawner(TileEntity te) {
+        if (te instanceof TileEntityMobSpawner) {
+            return true;
+        }
+
+        try {
+            Class<?> heeClass = Class.forName("chylex.hee.tileentity.TileEntityCustomSpawner");
+            if (heeClass.isInstance(te)) {
+                return true;
+            }
+        } catch (ClassNotFoundException ignored) {}
```

Comment:
I suggest using *optional* dependency on HEE and `Loader.isModLoaded` instead of reflection.

Reply:
I urge you not to look too closely at the rest of the class, then, because it is checking a LOT OF classes like this. 🤣 

That being said, looking at this, it looks like it's doing most of its class checks in a `static` block rather than at runtime. I will move the class check there so it at least only does it once per load.